cookbooks/aws-parallelcluster-config/spec/unit/resources/manage_raid_mount_unmount_spec.rb

```diff
@@ -0,0 +1,196 @@
+require 'spec_helper'
+
+class Chef::Resource::RubyBlock
+  def wait_for_block_dev(path)
+    # do nothing
+  end
+end
+
+def mock_get_raid_devices(devices)
+  allow_any_instance_of(Chef::Resource).to receive(:get_raid_devices).with("/dev/md0").and_return(devices)
```

Comment:
Nitpick. I would pass `/dev/md0` as parameter as well.

Reply:
Actually this is not needed anymore